scss/detail.scss

```diff
@@ -1,362 +1,321 @@
+          line-height: 24px;
         }
-  
-        .book-list-autor{
+        
+        .detail {
           color: $gray;
-          font-size: 14px;
-          line-height: 18px;
+          font-size: 20px;
+          line-height: 24px;
+          margin-bottom: 36px;
         }
       }
     }
-
-    .book-container:hover{
-      transition: width 2s, height 2s, transform 1s;
-      transform: scaleX(1.1) scaleY(1.1);
-    }
   }
-  .cover-text-container{
-    display: flex;
-    flex-direction: row;
-    height: 416px;
-    width: 900px;
-    background-color: $white;
-    box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
-  
-    .cover-page{
-      display: flex;
-      height: 368px;
-      width: 261px;
-      margin: 24px 52px 24px 24px;
+
+  @media only screen and (max-width: $tablet){
+    height: 768px;
+    width: 1024px;
+    grid-template-rows: 99px 569px 100px;
+    grid-template-areas:
+      "return-button"
+      "container"
+      ".";
+
+    .return-button {
+      font-size: 18px;
+      grid-area: return-button;
+      justify-self: flex-start;
+      margin: 0 0 33px 0;
     }
-  
-    .container-text{
-      display: flex;
-      flex-direction: column;
-      margin-top: 24px;
-    
-      .book-title{
-        color: $black;
-        font-size: 32px;
-        font-weight: 700;
-        line-height: 38px;
+
+    .cover-text-container {
+      height: 569px;
+      width: 853px;
+      grid-area: container;
+      justify-items: center;
+      grid-template-columns: 329px 524px; 
+      grid-template-rows: 71px 74px 368px 56px;
+      justify-self: center;
+      margin: 0;
+      grid-template-areas:
+        ". title"
+        ". genre"
+        "image description"
+        ". .";
+
+      .cover-page-badge-container {
+        display: grid;
+        align-self: center;
+        justify-self: center;
+        grid-area: image;
+        
+        .badge {
+          width: calc(#{$width-badge} * 0.7);
+          height: calc(#{$height-badge} * 0.7);
+          right: -16px;
+          top: -10px;
+        }
       }
-      
-      .title-border{
-        height: 4px;
-        width: 540px;
-        background-color: $earls-green;
-        margin-bottom: 36px;
+
+      .book-title {
+        margin: 0;
+        grid-area: title;
+        justify-self: flex-start;
+        align-self: flex-end;
       }
       
-      .title-genre{
-        color: $gray;
-        font-size: 24px;
-        font-weight: 700;
-        line-height: 28px;
-        margin-bottom: 12px;
-      }
-    }
-    
-    .container-secondary-text{
-      display: flex;
-      flex-direction: column;
-    
-      .secondary-title{
-        color: $black;
-        font-size: 20px;
-        font-weight: 700;
-        line-height: 24px;
+      .book-genre {
+        margin: 0 0 0 50px;
+        grid-area: genre;
+        justify-self: flex-start;
+        align-self: flex-start;
       }
+
+      .container-text {
+        display: grid;
```

Comment:
Is grid necessary here? There are no template-areas defined nor column/row dimensions :thinking: